samples/getting-started/ReactiveDemo/packages.config

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Newtonsoft.Json" version="6.0.4" targetFramework="net472" />
+  <package id="NuGet.Client" version="4.2.0" targetFramework="net472" />
+  <package id="NuGet.Common" version="4.2.0" targetFramework="net472" />
+  <package id="NuGet.Configuration" version="4.2.0" targetFramework="net472" />
+  <package id="NuGet.ContentModel" version="4.2.0" targetFramework="net472" />
+  <package id="NuGet.Frameworks" version="4.2.0" targetFramework="net472" />
+  <package id="NuGet.Packaging" version="4.2.0" targetFramework="net472" />
+  <package id="NuGet.Packaging.Core" version="4.2.0" targetFramework="net472" />
+  <package id="NuGet.Packaging.Core.Types" version="4.2.0" targetFramework="net472" />
+  <package id="NuGet.Protocol.Core.Types" version="4.2.0" targetFramework="net472" />
+  <package id="NuGet.Protocol.Core.v3" version="4.2.0" targetFramework="net472" />
```

Comment:
Convert to packagereference. 

See https://docs.microsoft.com/en-us/nuget/reference/migrate-packages-config-to-package-reference

It will allow us tricks like always targeting the latest ReactiveUI. Also some user issues have been fixed by using package reference. 

Make sure you make packagereference your default so you can do the migration

> If you go to Tools -> Options -> NuGet Package Manager -> General, You can change the "Default package management format" to "PackageReference". This should make that feature available to you.